dbt_subprojects/tokens/models/transfers_and_balances/zksync/tokens_zksync_base_transfers.sql

```diff
@@ -15,6 +15,7 @@
     traces = source('zksync','traces'),
     transactions = source('zksync','transactions'),
     erc20_transfers = source('erc20_zksync','evt_transfer'),
-    native_contract_address = '0x000000000000000000000000000000000000800a'
+    native_contract_address = '0x000000000000000000000000000000000000800a',
+    include_traces = false
```

Comment:
had to remove single quotes on this value to make it work (tested locally by running `dbt compile` to see raw sql)